GVRf/Framework/framework/src/main/java/org/gearvrf/utility/ResourceCache.java

```diff
@@ -40,7 +39,7 @@
  * 
  * @since 2.0.2
  */
-public class ResourceCache<T extends GVRObject> {
+public class ResourceCache<T extends GVRHybridObject> extends ResourceCacheBase {
```

Comment:
The issue with GVRHybridObject is that it assumes everything that is cacheable is a hybrid object, which can be too restrictive. However, we can address such issues separately. It's OK here.